5.0/en/0x02-Preface.md

```diff
@@ -1,57 +1,29 @@
 
 ## Introduction
 
-Established in 2008 through a collaborative community effort, the ASVS defines the security requirements to be considered when designing, developing, and testing modern web applications and web services.
+Originally launched in 2008 through a global community collaboration, the ASVS defines a comprehensive set of security requirements for designing, developing, and testing modern web applications and services.
 
-ASVS version 5.0 is the culmination of significant effort from the leaders, the working group, and other community members to update and improve this important standard.
+Following the release of ASVS 4.0 in 2019 and its minor update (v4.0.3) in 2021, Version 5.0 represents a significant milestone—modernized to reflect the latest advances in software security.
 
-Our goal for this version has been to make the ASVS easier to use while also making it more clearly focused on its defined scope and covering new, important areas of application development.
+ASVS 5.0 is the result of extensive contributions from project leaders, working group members, and the wider OWASP community to update and improve this important standard.
 
-## Key Objectives from ASVS Version 5.0
+## Principles behind version 5.0
 
-Version 5.0 has been developed with several key principles in mind.
+This major revision has been developed with several key principles in mind:
 
-### Clear Set of Requirements
+* Refined Scope and Focus: The standard now aligns more directly with the foundational pillars in its name: Application, Security, Verification, and Standard. Requirements have been rewritten to emphasize the prevention of security flaws rather than mandating specific technical implementations. Each requirement is now crafted to be self-explanatory, highlighting its rationale.
 
-The requirements for version 5.0 were prepared based on the following considerations:
+* Support for Documented Security Decisions: ASVS 5.0 introduces requirements for documenting key security decisions. This enhances traceability and supports context-sensitive implementations, allowing organizations to tailor their security posture to their specific specific needs and risks.
 
-* Aggressively deduplicate requirements to avoid controls or concepts being split across multiple places.
-* Clarify unclear or non-actionable requirement text.
-* Add new requirements to cover areas of particular concern, such as permissions, tokens, and cryptography.
-* Introduce new chapters and sections for areas that might not apply to all applications yet remain security-sensitive (e.g., OAuth and WebSockets).
+* Updated Levels: While ASVS retains its three-tier model, the level definitions have evolved to make the ASVS easier to adopt. Level 1 now serves as a foundational entry point focused on primary defenses. Level 2 represents a comprehensive viewo of standard security practices, and Level 3 addresses advanced, high-assurance requirements.
```

Comment:
would like to use "1st layer of defence" for Level 1, as this is the term we/I use everywhere for L1.

Remove "now" from "Level 1 now serves"

Reply:
Typo: "comprehensive viewo"

Reply:
How about:
"L1 is designed as the initial step to adopting the ASVS, providing the first layer of defense."